src/invidious/views/channel/about.ecr

```diff
@@ -0,0 +1,45 @@
+<% content_for "header" do %>
+<title><%= channel.author %> - Invidious</title>
+<link rel="stylesheet" href="/css/channel.css?v=<%= ASSET_COMMIT %>">
+<% end %>
+
+<% content_type =  5 %>
+<%= rendered "components/channels/channel-information" %>
+
+<div class="pure-g">
+    <% stats_style_append = ""%>
+    <% if !channel.country.empty? %>
+        <div class="pure-u-1 pure-u-md-3-4">
+            <div class="h-box">
+                <h3> <%= translate(locale, "channel_about_page_details_section_header") %> </h3>
+                <p>Country: <%=channel.country%></p> <!-- Translate Country Names -->
+            </div>
+        </div>
+    <% else %>
+        <% stats_style_append = "style=\"order: 1;\"" %>
+    <% end %>
+
+    <div class="pure-u-1 pure-u-md-1-4" <%= stats_style_append %>>
+        <div class="h-box">
+            <h3> <%= translate(locale, "channel_about_page_stats_section_header") %> </h3>
+            <p> <%= translate(locale, "channel_about_page_stats_joined_label") %>
+                <time datetime="<%=channel.joined.to_s("%Y-%m-%d")%>">
+                    <%= channel.joined.to_s(translate(locale, "%B %-d, %Y")) %>
```

Comment:
As long as we don't have a CLDR library to encode date properly, I think that it'd better to not translate them (and just use ISO format).